schema.sql

```diff
@@ -181,6 +181,7 @@ CREATE TABLE IF NOT EXISTS `remoteusers` (
   `actor_id` varchar(255) NOT NULL,
   `inbox` varchar(255) NOT NULL,
   `shared_inbox` varchar(255) NOT NULL,
+  `handle` varchar(255) DEFAULT '' NOT NULL,
```

Comment:
No need to add this to the schema files, as the migrations run on install and take care of this.

Reply:
Reverted to previous